client/state/sites/selectors/is-coming-soon-mode-active.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import { getSiteOption } from 'state/sites/selectors';
+
+/**
+ * Returns the ID of the coming soon page, or 0 if not set.
+ *
+ * @param {object} state Global state tree
+ * @param {object} siteId Site ID
+ * @returns {number} ID of the static page set as the coming soon page, or 0 if not set
+ */
+export default function isComingSoonModeActive( state, siteId ) {
+	return parseInt( getSiteOption( state, siteId, 'wpcom_public_coming_soon' ), 10 ) === 1;
```

Comment:
What's the difference between `wpcom_public_coming_soon` site option and the `is_coming_soon` value in the response from `/me/sites`? (I think that's the endpoint)